### PR TITLE
Update it-local.po

### PR DESCRIPTION
**Pull request by lordvinz**
Add Italian translation  for Addon:FamilyTree

### CODE REVIEW

File: FamilyTree/po/it-local.po [hunk 1]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1043
 msgid "Tree Options"
-msgstr ""
+msgstr "Opzioni per l'albero"
 
 #: FamilyTree/FamilyTree.py:1045
 msgid "Center Family"
-msgstr ""
+msgstr "Famiglia Principale"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: No camel case, that's only an English way of writing (and not the only one): "Famiglia principale"

File: FamilyTree/po/it-local.po [hunk 2]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1050
 msgid "The number of ancestor generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di antenati da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1053
 msgid "Descendant Generations"
-msgstr ""
+msgstr "Generazioni Discendenti"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: No camel case

File: FamilyTree/po/it-local.po [hunk 3]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1054
 msgid "The number of descendant generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di discendenti da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1057
 msgid "Kekule number of husband"
-msgstr ""
+msgstr "Numero kekule di mariti"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: Kekule uppercase (it's a surname)

File: FamilyTree/po/it-local.po [hunk 4]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1081
 msgid "Content"
-msgstr ""
+msgstr "Contenuto"
 
 #: FamilyTree/FamilyTree.py:1083
 msgid "Use call name"
-msgstr ""
+msgstr "Usa soprannomi"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: call name is translated as pseudonimo in gramps, please use the same translation (soprannome is used for nickname)

File: FamilyTree/po/it-local.po [hunk 5]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1085
 msgid "Don't use call name"
-msgstr ""
+msgstr "Non usare soprannomi"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: ditto

File: FamilyTree/po/it-local.po [hunk 6]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1086
 msgid "Replace first name with call name"
-msgstr ""
+msgstr "Sostituisci il nome con il soprannome"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: ditto

File: FamilyTree/po/it-local.po [hunk 7]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1087
 msgid "Underline call name in first name / add call name to first name"
-msgstr ""
+msgstr "Sottolinea il soprannome nel nome / aggiungi il soprannome al nome"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: ditto

File: FamilyTree/po/it-local.po [hunk 8]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1093
 msgid "Include Occupation"
-msgstr ""
+msgstr "Includi Occupazione"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase

File: FamilyTree/po/it-local.po [hunk 9]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1096
 msgid "Include Notes"
-msgstr ""
+msgstr "Includi Note"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase and no i with accent

File: FamilyTree/po/it-local.po [hunk 10]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1099
 msgid "Include Residence"
-msgstr ""
+msgstr "Includi Residenza"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase

File: FamilyTree/po/it-local.po [hunk 11]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1121
 msgid "Fall back to burial or cremation if death event missing"
-msgstr ""
+msgstr "Usa sepoltura o cremazione se manca l'evento morte"
 
 #: FamilyTree/FamilyTree.py:1124
 msgid "Protect private items"
-msgstr ""
+msgstr "Progetti gli item privati"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: item == elementi

File: FamilyTree/po/it-local.po [hunk 12]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:1133
 msgid "Include event description"
-msgstr ""
+msgstr "Includi descrizione dell'evento"
 
 #: FamilyTree/FamilyTree.py:1136
 msgid "Text Options"
-msgstr ""
+msgstr "Opzioni di Testo"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase

File: FamilyTree/po/it-local.po [hunk 13]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 
 #: FamilyTree/FamilyTree.py:907
 msgid "baptised"
-msgstr ""
+msgstr "battezzato"
 
 #: FamilyTree/FamilyTree.py:909
 msgid "died"
-msgstr ""
+msgstr "morto"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: maybe better "deceduto"?

File: FamilyTree/po/it-local.po [hunk 14]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 #: FamilyTree/FamilyTree.py:668 FamilyTree/FamilyTree.py:673
 msgid "Unknown"
-msgstr ""
+msgstr "Sconosciuto"
 
 #: FamilyTree/FamilyTree.py:675
 #, python-format
 msgid "%(father)s and %(mother)s"
-msgstr ""
+msgstr "%(padre)s e %(madre)s"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: like for other replacement markers, please keep them as they are: "%(father) e %(mother)s"

File: FamilyTree/po/it-local.po [hunk 15]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
 #: FamilyTree/FamilyTree.gpr.py:4
 msgid "Produces a graphical family tree."
-msgstr ""
+msgstr "Genera un albero genealogico grafico"
 
 #: FamilyTree/FamilyTree.py:102
 #, python-format
 msgid "Family Tree for %s"
-msgstr ""
+msgstr "Albero Genealogico per %s"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase

File: FamilyTree/po/it-local.po [hunk 16]
```diff
@@ -19,291 +19,292 @@ msgstr ""
 
 #: FamilyTree/FamilyTree.gpr.py:3
 msgid "Family Tree"
-msgstr ""
+msgstr "Albero Genealogico"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: no camelcase

File: FamilyTree/po/it-local.po [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Vincenzo Alfano <vinc.alfano@gmail.com>, YEAR.
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: Please also the year (if you use a program like poedit or lokalize it should update those headers automatically)

**Comment by lordvinz** [comment]
Done!

V

**Comment by tosky** [comment]
Can you please merge the two commits, so that only one commit will be pushed? (look for git interactive rebasing, https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

**Comment by lordvinz** [comment]
Dear Luigi,

> Il giorno 27 dic 2016, alle ore 23:27, Luigi Toscano <notifications@github.com> ha scritto:
> 
> Can you please merge the two commits, so that only one commit will be pushed? (look for git interactive rebasing, https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

I have no idea how to do this. I’m working from the browser, while the guide you sent me assume a terminal, I think (it keep telling, press s to squash, I have no idea where to press it). I’m just using the browser. Can you please give me an hint about how to merge the (now three) commits. I have also added Diego suggestions.

V

File: FamilyTree/po/it-local.po [hunk 18]
```diff
@@ -19,291 +19,292 @@ msgstr ""
...
-msgstr ""
+msgstr "Numero Kekule di mariti"
 
 #: FamilyTree/FamilyTree.py:1058
 msgid ""
 "The Kekule number of the husband (central family). Set 0 to not show Kekule "
 "numbers"
-msgstr ""
+msgstr "Il numero Kekule del marito (famiglia principale). Scegli 0 per non mostrarlo "
+"numbers"
```
Comment (tosky) [inline FamilyTree/po/it-local.po]: "numbers " should be removed now, and the space at the end of the previous line (after mostrarlo) should be removed as well.

**Comment by tosky** [comment]
Uhm, unless the administrator enabled the squashing of the commits (https://github.com/blog/2141-squash-your-commits) which is not a good thing in general, as you want to preserve the individual commits, I fear that there is no way to do it from the browser. You would probably need a Git client, from the command line or graphical.

**Comment by Nick-Hall** [comment]
@tosky Squash merging is enabled for this repository.  I have added you to the _Addon Developers_ team.

**Comment by tosky** [comment]
@Nick-Hall oh, thanks! I see now that the squashing is one of the possible choices. I will wait for the last change that I requested and then I will merge.

**Comment by lordvinz** [comment]
Done! Just didn’t squased, since I still don’t understand how.

V

**Comment by tosky** [comment]
Thanks; I did the squashing thanks to github. I would recommend a bit of git training for the future anyway :)

**Comment by sam-m888** [comment]
Thank you

Addon rebuilt for Gramps master:
https://github.com/gramps-project/addons/commit/a0689ca73501475bcec2bebd725593390a472745
https://github.com/gramps-project/addons-source/commit/57f882140d72f6cdc8263b8f1b24f95ad69b1225

**Pull request merged**